apps/cli/src/commands/project/import.project.ts

```diff
@@ -0,0 +1,182 @@
+import type {
+  CommandActionData,
+  CommandArgument,
+  CommandOption
+} from '@/types/command/command.types'
+import BaseCommand from '../base.command'
+import { confirm, text } from '@clack/prompts'
+import ControllerInstance from '@/util/controller-instance'
+import { Logger } from '@/util/logger'
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import dotenv from 'dotenv'
+import secretDetector from '@keyshade/secret-scan'
+
+export default class ImportFromEnv extends BaseCommand {
+  getName(): string {
+    return 'import'
+  }
+
+  getDescription(): string {
+    return 'Imports environment secrets and variables from .env file to a project.'
+  }
+
+  getArguments(): CommandArgument[] {
+    return [
+      {
+        name: '<Project Slug>',
+        description: 'Slug of the project where envs will be imported.'
+      }
+    ]
+  }
+
+  getOptions(): CommandOption[] {
+    return [
+      {
+        short: '-f',
+        long: '--env-file <string>',
+        description: 'Path to the .env file'
+      }
+    ]
+  }
+
+  canMakeHttpRequests(): boolean {
+    return true
+  }
+
+  async action({ args, options }: CommandActionData): Promise<void> {
+    const [projectSlug] = args
+
+    try {
+      const { envFilePath } = await this.parseOptions(options)
+      if (!envFilePath) return
```

Comment:
This case would create ambiguity in the near future. If you have any specific explanation for adding this, then do include that in comments. I think its best to not keep this here, and do all the validation in `parseOptions`. That function should return a definite `envFilePath` thats non null and non empty